shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java

```diff
@@ -104,9 +103,9 @@ public HttpClient httpClient(final HttpClientProperties properties) {
                     }
                     // The write and read timeouts are serving as generic socket idle state handlers.
                     tcpClient = tcpClient.doOnConnected(connection -> {
+                        connection.addHandlerLast(new IdleStateHandler(properties.getReaderIdleTime(), properties.getWriterIdleTime(), properties.getAllIdleTime(), TimeUnit.MILLISECONDS));
```

Comment:
why add this same？

Reply:
I have removed the redundant code

Reply:
> why add this same？

He is trying to fix https://github.com/apache/incubator-shenyu/issues/2882. I think even if you remove this configuration, there should have another default configuration, @zouchangfu

Reply:
I found the default configuration in `org.springframework.http.client.Netty4ClientHttpRequestFactory#readTimeout`, is that correct ? @zouchangfu

Reply:
I don't think no limit is a good idea. Maybe it's better to limit the timeout that users configure too much ?

Reply:
Of course, also need to set a larger `readTimeout`.